lib/pprof/pprof.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pprof
+
+import (
+	"fmt"
+	"net/http"
+	_ "net/http/pprof"
+	"net/url"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+type PprofPort int
+
+const basePort = 6060
+
+const (
+	VchInitPort PprofPort = iota
+	ImagecPort
```

Comment:
#1238 is not going in yet - @jzt has hit some issues that need resolving. I'm not sure it'll go in for TP3, so let's leave this as is for now unless it has the potential to cause imagec functional issues - @caglar10ur ?